src/types/Criteria.cpp

```diff
@@ -1,5 +1,5 @@
-#include "Criteria.h"
-#include "Criterion.h"
+#include "../../include/types/Criteria.h"
+#include "../../include/types/Criterion.h"
 #include <algorithm>
 #include <iostream>
```

Comment:
should we start using static variable to make the include header more readable ? 
Something that would look like this ?
ie. path_to_type = ""../../include/types/"
--> #include path_to_type+"Criteria.h"

Reply:
I don't know how I feel about it, the path aren't that complicated for the moment and it should not really get much worse than that and it stays more readable! But if we get to a point where we have looooooong import path that we often use, let's do that 👍